### PR TITLE
Prevent swarmers from destroying active clone pods

### LINKED ISSUE #9160: Swarmers and Cloners
**Problem Description**:
Swarmers are able to eat the Cloning machine while someone is currently being cloned, causing a forced eject of the person catatonic and genetically messed up

**What did you expect to happen**: 
Swarmers should not be able to eat a cloner while someone is cloning, it goes against their laws to kill someone

**What happened instead**:
i was unable to clone, my body was catatonic

**Why is this bad/What are the consequences:**
it goes against their law sets and ruins rounds

**Steps to reproduce the problem**:
my body was put in the cloner, and i began to regenerate, during the process a swarmer came up and dissasembled the cloner

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: i cannot recall any more details about the round time, all i was doing was ghosting waiting for my body to form

### PR DESCRIPTION
**Pull request by alexkar598**
also decapitalize the S in sentient in swarmies laws.

Fixes #9160 

:cl:alexkar598
tweak:Swarmers can no longer destroy active clone pods.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/miniantags/bot_swarm/swarmer.dm [hunk 1]
```diff
@@ -306,6 +306,11 @@
 /obj/spacepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Destroying this vehicle would destroy us. Aborting.</span>")
 
+/obj/machinery/clonepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(attempting)
+		to_chat(S, "<span class='warning'>Destroying this machine while an it is occupied would result in biological and sentient resources to be harmed. Aborting.</span>")
```
Comment (Citinited) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: Remove the `an` from `while an it is occupied `

Comment (alexkar598) [reply code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: errr,dumb me

Comment (alexkar598) [reply code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: Grammer addressed

File: code/game/gamemodes/miniantags/bot_swarm/swarmer.dm [hunk 2]
```diff
@@ -306,6 +306,11 @@
 /obj/spacepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Destroying this vehicle would destroy us. Aborting.</span>")
 
+/obj/machinery/clonepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(attempting)
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: why not check for `occupant`?

Comment (Crazylemon64) [reply code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: Please make the change within two days or this PR shall be closed for staleness.

**Comment by alexkar598** [comment]
May I ask why use occupant instead of attempting? The cloner specifically made a var for this exact purpose and you want to not use it...because you can?

**Comment by Crazylemon64** [comment]
https://github.com/ParadiseSS13/Paradise/blob/master/code/game/machinery/cloning.dm#L287
It's unset at the beginning of the clone cycle - your change won't work as you'll want it to

**Comment by alexkar598** [comment]
Attempting is set before occupant FYI

**Comment by RyanSmake** [comment]
Can confirm, attempting is set before occupant.

**Comment by Fox-McCloud** [comment]
No change in two days; closing. Someone else can submit a fix for this, if they like.

**Pull request closed without merging**